sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_Healthcare.cs

```diff
@@ -0,0 +1,75 @@
+
+            AnalyzeHealthOperation healthOperation = client.StartAnalyzeHealth(document);
+
+            await healthOperation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection results = healthOperation.Value;
+
+            Console.WriteLine($"Results of Azure Text Analytics \"Healthcare\" Model, version: \"{results.ModelVersion}\"");
+            Console.WriteLine("");
+
+            foreach (RecognizeHealthcareEntititesResult result in results)
+            {
+                if (result.HasError)
+                {
+                    Console.WriteLine($"    Document error code: {result.Error.ErrorCode}.");
+                    Console.WriteLine($"    Message: {result.Error.Message}.");
+                }
+                else
+                {
+                    Console.WriteLine($"    Recognized the following {result.Entities.Entities.Count} healthcare entities:");
+
+                    foreach (HealthcareEntity entity in result.Entities.Entities)
+                    {
+                        Console.WriteLine($"    Entity: {entity.Text}");
+                        Console.WriteLine($"    Subcategory: {entity.Subcategory}");
+                        Console.WriteLine($"    Offset: {entity.Offset}");
+                        Console.WriteLine($"    Length: {entity.Length}");
+                        Console.WriteLine($"    IsNegated: {entity.IsNegated}");
+                        Console.WriteLine($"    Links:");
+
+                        foreach (HealthcareEntityLink healthcareEntityLink in entity.Links)
+                        {
+                            Console.WriteLine($"        ID: {healthcareEntityLink.Id}");
+                            Console.WriteLine($"        DataSource: {healthcareEntityLink.DataSource}");
+                        }
+                    }
+
+                    Console.WriteLine($"    Document statistics:");
+                    Console.WriteLine($"        Character count (in Unicode graphemes): {result.Statistics.CharacterCount}");
+                    Console.WriteLine($"        Transaction count: {result.Statistics.TransactionCount}");
+                    Console.WriteLine("");
```

Comment:
This values are empty here because we are not asking for statistics right? if so, we could delete from this sample